apiextensions/storageversion/cmd/migrate/main.go

```diff
@@ -50,9 +56,15 @@ func main() {
 		logger.Fatal(err)
 	}
 
+	ignoreNotFound, err := strconv.ParseBool(os.Getenv(IgnoreNotFound))
+	if err != nil {
+		logger.Infof("Error getting environment variable %s for migrate %w", IgnoreNotFound, err)
+	}
+
```

Comment:
ignore the error since the result will return `false` in that instance. By default we don't expect this to be set

```suggestion
	ignoreNotFound, _ := strconv.ParseBool(os.Getenv(IgnoreNotFound))

```